os/include/tinyara/net/if/wifi.h

```diff
@@ -27,6 +27,9 @@
 #define TRWIFI_PASSPHRASE_LEN		  64
 #define TRWIFI_CBK_MSG_INITIALIZER {0, {0,}, NULL}
 
+/*  reason */
+#define TRWIFI_REASON_UNKNOWN 0
```

Comment:
@pillip8282 I can't find where this definition is used.

Reply:
vdev_handler.c uses this value to test correctness of event information from wi-fi driver.
More definition will be added later.